clients/python/test/e2e/test_notebooks.py

```diff
@@ -0,0 +1,33 @@
+import papermill as pm
```

Comment:
is `pm` the usual shortcut? (as `np` or `sa`?)  Otherwise, I would prefer `papermill` in here

Reply:
It's what'd done in the examples/tutorials at least: https://papermill.readthedocs.io/en/latest/usage-execute.html#execute-via-the-python-api
So I believe it is somewhat standard.